### DISCUSSION TITLE
Using 'Read the docs' instead of github wiki pages

### DISCUSSION THREAD

**Discussion by JojoS62** [mbed-ce/mbed-os#198] [Ideas]
Several projects are using this documentation pages:
https://about.readthedocs.com/
It is better structured than the simple github Wiki pages, how about using readthedocs? 
There is a community edition, but it looks like you need an own domain/server for this.

**Comment by multiplemonomials** [comment]
Docs are a difficult issue... there are both the doxygen docs, and also the `mbed-os-5-docs` repo's docs.  It seems like Mbed had some sort of tooling which would integrate the two together, embedding the doxygen documentation inside the handwritten pages.
